www/src/css/snaplist.css

```diff
@@ -34,6 +44,10 @@
   margin-top:140%;
 }
 
+.b-store .b-snaplist_grid .b-snaplist__item:before {
```

Comment:
The nesting sucks but it seems the lesser of 2 evils at this stage. There's a couple of comments already in `snaplist.css` about _grid style_ and _row style_ though there could be some additional explanation.

The additional nesting comes into play here as the snap cards on the store page are taller than on the home page due to having the install button and price. Marionette's nested views are *then* involved which makes the situation a bit more convoluted.